data/remnant missions.txt

```diff
@@ -1252,3 +1252,240 @@ mission "Remnant: Expanded Horizons Quarg 3"
+			label end
+			`	"Next time you find one of these drives, just check with a starport director. They will know where it will need to go. A word of warning though; I strongly recommend not using these drives - there may disappear at the most inconvenient of times, and there won't always be an outfitter on hand." He turns and heads off with the rest of the team, turning briefly to give you friendly wave as they disembark.`
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+
+
+
+mission "Remnant: Broken Jump Drive 2"
+	name "Broken Jump Drive Delivery"
+	description "The researchers on <planet> have requested your broken jump drive."
+	to offer
+		has "Remnant: Broken Jump Drive 1: done"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		attributes outfitter
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`Remembering the advice given by the engineer who paid you for the broken jump drive, you stop by the starport director's desk. Information must flow quickly amongst the Remnant, because by the time you arrive at the director's desk she already has the information ready for you.`
+			`	"Greetings, Newcomer," she chants. "Are you interested in turning in the broken jump drive you salvaged to our researchers?"`
+			choice
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"They will definitely appreciate new material to research. The team is located on <planet>. I will send a notification ahead so they will be expecting you."`
+				accept
+			label refuse
+			`	She looks at you thoughtfully and sings, "It is unlikely that anyone else would pay for it, but it is your choice. Return if you change your mind."`
+				defer
+	on complete
+		conversation
+			`As promised, there is a team of researchers waiting when you settle onto the designated landing pad. One comes over to hand you <payment> while the rest carefully unload the broken drive and start hauling it back to their lab.`
+			`	"Our thanks, Captain <last>. Having a partially functional drive like this will be very beneficial to our efforts." His voice cracks a bit, as if unused to being used much.`
+			choice
+				`	"Great! I will find more."`
+					goto end
+				`	"What are you studying?"`
+			`	"Our previous research into Jump Drives determined that these drives are unsurprisingly multi-part systems. We know the outer part that we can see and handle is mostly the machinery for generating some kind of containment field or pocket. As to how it works or why it is needed..." his chant trails off and he makes a gesture you suspect may be the equivalent of a shrug. "Anyways, our team is focused on the nature of this outer system. It is our hope that - damaged as it is - its continued functionality might allow us to look a little farther inside its inner workings to figure out exactly what that field does and how it is used."`
```

Comment:
Lower case jump drives.
Perhaps drop the hyphens for commas, since they aren't proper em dashes? `our hope is that, damaged as it is, its continued...`
Drop the ending quotation mark.